pom.xml

```diff
@@ -1294,6 +1359,11 @@
       </dependency>
 
       <!-- Third-party -->
+      <dependency>
+        <groupId>com.conversantmedia</groupId>
+        <artifactId>disruptor</artifactId>
+        <version>1.2.15</version>
```

Comment:
I see you went back a few versions in the last commit, but I'm confused, 1.2.19 depends on slf4j just like 1.2.15: https://mvnrepository.com/artifact/com.conversantmedia/disruptor/1.2.15
(even the older 1.2.13 had that dependency)

Reply:
Yes I found the research from @fgdrf and eventually just added an exclude